.travis.yml

```diff
@@ -1,26 +1,36 @@
-matrix:
+jobs:
   include:
-  - name: 'rspec Tests'
-    language: ruby
-    rvm:
-      - 2.3
-    cache: bundler
-    before_install:
-      - gem update --system
-      - gem install bundler
-    script:
-      - bundle exec rake
-      - bundle exec rake rubocop
-  - name: "Integration Tests"
-    language: node_js
-    node_js: node
-    cache: yarn
-    before_install:
-      - rvm use 2.3 --install --binary --fuzzy
-      - ruby --version
-      - gem update --system
-      - gem install bundler
-      - bundle install --jobs=3 --retry=3 --deployment
-    script:
-      - yarn test
-      - yarn lint
+    - stage: 'Test'
+      name: 'rspec Tests'
+      language: ruby
+      rvm:
+        - 2.3
+      cache: bundler
+      before_install:
+        - gem update --system
+        - gem install bundler
+      script:
+        - bundle exec rake
+        - bundle exec rake rubocop
+    - name: "Integration Tests"
+      language: node_js
+      node_js: node
+      cache: yarn
+      before_install:
+        - rvm use 2.3 --install --binary --fuzzy
+        - ruby --version
+        - gem update --system
+        - gem install bundler
+        - bundle install --jobs=3 --retry=3 --deployment
+      script:
+        - yarn test
+        - yarn lint
+    # TODO: Only add this stage on master?
```

Comment:
It probably makes sense to only add this stage on master. I think you can add this:
```
branches:
  only:
  - master
```

Reply:
Done!